lib/patches/before/openssl.rb

```diff
@@ -0,0 +1 @@
+require "openssl-stubs" unless ENV['TRUFFLERUBY_OPENSSL_ENABLED']
```

Comment:
I updated this to: `require "openssl-stubs" if ENV['TRUFFLERUBY_OPENSSL_DISABLED']` so it fails by default.